### ISSUE TITLE
camera still on after disconnect.

### ISSUE THREAD

**Issue by vijaywayu2014** [twilio/video-quickstart-js#53]
when I disconnect call it detach the remote and local tracks and release the view but camera is not turning off. 
var token     = resp.call_token;
        var room_name = resp.room_name;
        //creating local tracks and join room
        var videoWidth = $("#chatwindowdiv").width();
        console.log("Joining room '" + $scope.room_name + "'...");
        var connectOptions = {
          name: $scope.room_name,
          //tracks: previewTracks,
          //logLevel: 'debug'
        };

        if (previewTracks) {
          connectOptions.tracks = previewTracks;
        }

        // Join the Room with the token from the server and the
        // LocalParticipant's Tracks.
        Video.connect(token, connectOptions).then(roomJoined, function(error) {
          console.log('Could not connect to Twilio: ' + error.message);
        });
        $scope.roomDisconnect= function() {
          console.log('Leaving room...');
          activeRoom.disconnect();
        };
function roomJoined(room) {
  window.room = activeRoom = room;

  console.log("Joined as '" + identity + "'");
  // Attach LocalParticipant's Tracks, if not already attached.
  var previewContainer = document.getElementById('local-media');
  if (!previewContainer.querySelector('video')) {
    attachParticipantTracks(room.localParticipant, previewContainer);
  }

  // Attach the Tracks of the Room's Participants.
  room.participants.forEach(function(participant) {
    console.log("Already in Room: '" + participant.identity + "'");
    var previewContainer = document.getElementById('remote-media');
    attachParticipantTracks(participant, previewContainer);
  });

  // When a Participant joins the Room, log the event.
  room.on('participantConnected', function(participant) {
    console.log("Joining: '" + participant.identity + "'");
  });

  // When a Participant's Track is subscribed to, attach it to the DOM.
  room.on('trackSubscribed', function(track, publication, participant) {
    console.log("Subscribed to " + participant.identity + "'s track: " + track.kind);
    console.log(participant);
    var videoWidth = $("#chatwindowdiv").width();
    document.getElementById('remote-media').style.width=videoWidth+"px";
    document.getElementById('local-media').style.width="210px";
    var previewContainer = document.getElementById('remote-media');
    attachTracks([track], previewContainer);
  });

  // When a Participant's Track is unsubscribed from, detach it from the DOM.
  room.on('trackUnsubscribed', function(track, publication, participant) {
    console.log("Unsubscribed from " + participant.identity + "'s track: " + track.kind);
    $scope.call_inprogress  = false;
    console.log(track);
    detachTracks([track]);
  });

  // When a Participant leaves the Room, detach its Tracks.
  room.on('participantDisconnected', function(participant) {
    console.log("RemoteParticipant '" + participant.identity + "' left the room");
    $('#button-leave').addClass('ng-hide');
    $scope.call_inprogress  = false;
    if (previewTracks) {
      console.log('inside preview tracks');
      previewTracks.forEach(function(track) {
        track.stop();
        var detachedElement = track.detach();
        detachedElement.forEach(element => element.remove());
      });
    }
    console.log(room.localParticipant);
    console.log(participant);
    detachParticipantTracks(room.localParticipant);
    detachParticipantTracks(participant);
  });

  // Once the LocalParticipant leaves the room, detach the Tracks
  // of all Participants, including that of the LocalParticipant.
  room.on('disconnected', function() {
    console.log('Left');
    if (previewTracks) {
      previewTracks.forEach(function(track) {
        track.stop();
      });
    }
    detachParticipantTracks(room.localParticipant);
    room.participants.forEach(detachParticipantTracks);
    activeRoom = null;
  });
}

**Comment by manjeshbhargav** [comment]
Hi @vijaywayu2014 ,

Thank you for your question. Are you using `twilio-video.js@2.0.0-beta1`? If so, we are working on fixing a bug in this version where the camera lights are on even after calling `LocalVideoTrack#stop()`.
We will reach out to you as soon as a fix is available.

Please watch this space for updates.

Thanks,

Manjesh Malavalli
JSDK Team

**Comment by manjeshbhargav** [comment]
Hi @vijaywayu2014 ,

We have a Release Candidate available that fixes the behavior you are seeing. Can you try it out and let us know if it works for you?

## NPM
```
npm install twilio/twilio-video.js#2.0.0-rc3
```

## <script>
```
<script src="//stage.twiliocdn.com/sdk/js/video/releases/2.0.0-rc3/twilio-video.js">
```

Thanks,

Manjesh Malavalli
JSDK Team

**Comment by vijaywayu2014** [comment]
The Problem resolved by including <script src="//stage.twiliocdn.com/sdk/js/video/releases/2.0.0-rc3/twilio-video.js"> in your index file.

**Comment by manjeshbhargav** [comment]
@vijaywayu2014 ,

We're happy that the latest RC resolved the issue. We will formally release `twilio-video.js@2.0.0-beta2` with these changes soon.

Thanks,

Manjesh Malavalli
JSDK Team
